src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -143,6 +147,42 @@ struct DisallowedParamsInserter {
   mobile_apis::VehicleDataResultCode::eType code_;
 };
 
+ResponseInfo::ResponseInfo()
```

Comment:
@LuxoftAKutsan 
Why pass application manager reference? Application manager is member of base class.